### PR TITLE
ci(source): add ci test

### PR DESCRIPTION
**Pull request by waruto210**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

Add a test in ci for parsing the very complex protobuf schema file: `metabit.proto`.

## Checklist

~- [ ] I have written necessary rustdoc comments~
~- [ ] I have added necessary unit tests and integration tests~
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

**Comment by skyzh** [comment]
Is it okay to put it in the public repo?

**Comment by skyzh** [comment]
... at least where it is from should be anonymous?

File: ci/scripts/e2e-source-test.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ echo "--- Download mise"
 buildkite-agent artifact download avro-simple-schema.avsc ./
 buildkite-agent artifact download avro-complex-schema.avsc ./
 buildkite-agent artifact download proto-complex-schema ./
+buildkite-agent artifact download proto-metabit-assembly-schema ./
```
Comment (skyzh) [inline ci/scripts/e2e-source-test.sh:35]: These 4 files are uploaded in the build stage but only used in e2e-source-test stage. Why not directly use the one in the repo / do local copy?

**Review by skyzh — APPROVED** [review]
Rest LGTM. If you need to hide this PR from the public, I can help you contact GitHub support.

**Comment by waruto210** [comment]
> 

Sorry, I didn't consider if this schema file can be published, this needs to be consulted with @neverchanje , we may hide it first.

**Comment by waruto210** [comment]
> Is it okay to put it in the public repo?

Sorry, I didn't consider if this schema file can be published, this needs to be consulted with @neverchanje , we may hide it first.

**Comment by skyzh** [comment]
Okay, I'll hide this PR.

**Pull request closed without merging**